docs/sitemap.txt

```diff
@@ -86,6 +86,7 @@ index.md
 		Shipping-prebuilt-binaries-as-wraps.md
 		fallback-wraptool.md
 	Release-notes.md
+		Release-notes-for-0.59.0.md
```

Comment:
You should not add this here. The web site is autogenerated and deployed on every merge to master so this would go out immediately. Instead the content needs to go to a file `docs/markdown/snippets/newvsbackends.md`. When the release happens all snippets are merged into the final file automatically.

Reply:
Hi @jpakkane,

Shifted the items accordingly--I hope I understood things correctly.  I also fixed the formatting a bit in the update to `BuiltinOptions.md` file.